DOCS/canonical-identity-platform-tutorial-deploy-identity-bundle.md

```diff
@@ -1,191 +1,209 @@
-> - you have a client registered in Entra ID; you need to know the `client_id`, `client_secret` and `tenant_id`.
+Unit         Workload  Agent  Address      Ports  Message
+hydra/0*     active    idle   10.1.57.88          
+kratos/0*    active    idle   10.1.57.102         
+login-ui/0*  active    idle   10.1.57.92          
 
-> See more: [Microsoft | Register a new application](https://learn.microsoft.com/en-us/azure/healthcare-apis/register-application#register-a-new-application), [Microsoft | Register a new application > Certificates & Secrets](https://learn.microsoft.com/en-us/azure/healthcare-apis/register-application#certificates--secrets) and [Microsoft | How to find your Microsoft Entra tenant ID](https://learn.microsoft.com/en-us/entra/fundamentals/how-to-find-tenant)
+Offer              Application  Charm   Rev  Connected  Endpoint     Interface    Role
+kratos-info-offer  kratos       kratos  527  0/0        kratos-info  kratos_info  provider
+oauth-offer        hydra        hydra   362  0/0        oauth        oauth        provider
+```
 
+# Use the built-in Identity Provider
 
-[note]
+The Identity Platform comes with a built-in identity provider,
+allowing you to manage user accounts internally without relying on a third-party service.
+You can also let your users authenticate both with external providers and the local one.
 
-If you don't have a Microsoft Entra ID tenant, you can try out the Identity Platform in identity provider mode.
-To do that, proceed to the next tutorial section.
+The local identity provider feature is enabled by default in Charmed Kratos.
 
-[/note]
+At this stage, you can create your personal admin account:
 
-In this tutorial we are going to connect the Identity Platform with Microsoft Entra ID.
+```shell
+juju run kratos/0 create-admin-account email=<your-email> username=<username>
+```
 
-First, we need to inform Entra ID of our deployment’s `redirect_uri`. Get the Kratos base URL:
+After the account is created, use the provided password reset link to set your password and complete the setup.
 
-```
-juju run traefik-public/0 show-proxied-endpoints | yq '.proxied-endpoints' | jq '.kratos.url'
-```
+Congratulations, you're all set to use the built-in identity provider to sign in.
 
-The `redirect_uri` should be: `<kratos-base-url>/self-service/methods/oidc/callback/microsoft`. Use this to configure your client in Entra ID.
+# Connect the Canonical Identity Platform with an external Identity Provider
 
-> See more: [Microsoft | Redirect URI](https://learn.microsoft.com/en-us/entra/identity-platform/reply-url)
+To use an external identity provider (e.g. Google, Entra ID, GitHub...) alongside the local one,
+you can leverage [Kratos External IdP Integrator](https://charmhub.io/kratos-external-idp-integrator).
 
-Now use the Entra ID client credentials to configure the Kratos external IDP integrator charm:
+> This part of the tutorial assumes that you have a GitHub account.
```

Comment:
This feels like it should come right beneath the section title, as it's not really meaningful for people to read anything in this section if they don't meet that prerequisite.